### ISSUE TITLE
Global configuration

### ISSUE THREAD

**Issue by lolo101** [lolo101/MsgViewer#151]
The application can be configured with user configuration file in $HOME/.MSGViewer

It would be nice to have a global configuration file under /etc/MSGViewer for cases where the application is not user-specific

**Comment by lolo101** [comment]
The global configuration is a fallback for when the local configuration is missing
The default configuration is a fallback when the global configuration is missing

**Comment by lolo101** [comment]
For Windows platforms, the configuration files should be located in
%LOCALAPPDATA%\MSGViewer
%ALLUSERSPROFILE%\MSGViewer

**Comment by lolo101** [comment]
Work on windows Vista - 11

**Comment by sbasmanov** [comment]
Hi @lolo101
Is it available for linux systems? I tried /etc/MSGViewer without success. I need to set default mode to HTML (RTFFormat=no)
Thanks

**Comment by lolo101** [comment]
Hi @sbasmanov and thank you for your interest in the project

This issue is still a feature in development and is not yet available.

On Linux, the configuration is currently located under ~/.MSGViewer

**Comment by sbasmanov** [comment]
> On Linux, the configuration is currently located under ~/.MSGViewer

Yes, I know. To be precise, MSGViewer.properties. But I need to set this option globally, for all users (this is terminal server). I haven't found in source - is it possible to supply command line arguments when starting app? This way I can make shell script that will start app with proper arguments. I tried -DRTFFormat=no but it doesn't work. For now I changed source to always select HTML radio button, but would be great to use original package with config or command line option.
Thanks.

**Comment by lolo101** [comment]
@sbasmanov For the moment it is not possible to set options globally.
However, for your specific need, you can start the application with the "-starttextview" argument
For instance, this should do the trick:
`java -jar msgviewer.jar -starttextview`

Let me know if this approach works

**Comment by sbasmanov** [comment]
> However, for your specific need, you can start the application with the "-starttextview" argument

Great. I'll try this on Monday and let You know if that worked. Thanks.

**Comment by sbasmanov** [comment]
> However, for your specific need, you can start the application with the "-starttextview" argument

That did the trick :) Thank You very much.
